lib/std/os.zig

```diff
@@ -99,6 +99,7 @@ pub const MADV = system.MADV;
 pub const MAP = system.MAP;
 pub const MSF = system.MSF;
 pub const MAX_ADDR_LEN = system.MAX_ADDR_LEN;
+pub const MAXNAMLEN = system.MAXNAMLEN;
```

Comment:
Is it a problem that `MAXNAMLEN` isn't defined in all `system` definitions? (`MAXNAMLEN` is a BSD-specific name)

If so, then it might make more sense to change this line:

https://github.com/ziglang/zig/blob/d3b1cdf508242cbef9b107aaf5ede627e63c9e7b/lib/std/fs.zig#L60

to

```zig
    .netbsd, .openbsd, .solaris => os.system.MAXNAMLEN,
```

which should fix https://github.com/ziglang/zig/issues/13680 as well (which I introduced in 13153 😞)

Reply:
It is indeed only defined on a few systems but I didn't think that would be a problem considering that you would only use it where you actually need it (on systems that have that constant), but it does look like accessing `os.system` like that is a pattern that exists as well so it seems like a better solution really.